### PR TITLE
xmlto: upgrade to a new release

### LINKED ISSUE #26442: FTBFS xmlto
```
2024/08/15 21:07:58 WARN xmlif/xmlif.l:46:8: error: type defaults to 'int' in declaration of 'ifsense' [-Wimplicit-int]
2024/08/15 21:07:58 WARN    46 | static ifsense;                 /* sense of last `if' or unless seen */
2024/08/15 21:07:58 WARN       |        ^~~~~~~
2024/08/15 21:07:58 WARN xmlif/xmlif.l:243:1: error: return type defaults to 'int' [-Wimplicit-int]
2024/08/15 21:07:58 WARN   243 | {
2024/08/15 21:07:58 WARN       | ^   
2024/08/15 21:07:58 WARN xmlif/xmlif.c:2002:16: warning: 'input' defined but not used [-Wunused-function]
2024/08/15 21:07:58 WARN  2002 |     static int input  (void)
2024/08/15 21:07:58 WARN       |                ^~~~~
2024/08/15 21:07:58 WARN xmlif/xmlif.c:1961:17: warning: 'yyunput' defined but not used [-Wunused-function]
2024/08/15 21:07:58 WARN  1961 |     static void yyunput (int c, register char * yy_bp )
2024/08/15 21:07:58 WARN       |                 ^~~~~~~
2024/08/15 21:07:58 WARN make[1]: *** [Makefile:741: xmlif/xmlif.o] Error 1
2024/08/15 21:07:58 INFO make[1]: Leaving directory '/home/build'
2024/08/15 21:07:58 INFO make: Leaving directory '/home/build'
2024/08/15 21:07:58 WARN make: *** [Makefile:584: all] Error 2
```

### PR DESCRIPTION
**Pull request by xnox**
Wow new release of xmlto. Update to git-checkout from pagure.io.

Fixes: #26442

### CODE REVIEW

**Review by rawlingsj — CHANGES_REQUESTED** [review]
the package update check is failing because it is still pointing at release monitoring which is still tracking 0.0.28 as the latest version.  You can use the new `git:` config now which I just tested and worked

```
update:
  enabled: true
  git: {}
```

**Comment by xnox** [comment]
> the package update check is failing because it is still pointing at release monitoring which is still tracking 0.0.28 as the latest version. You can use the new `git:` config now which I just tested and worked
> 
> ```
> update:
>   enabled: true
>   git: {}
> ```

but this schema is not yet in melange? so we can't use it yet in production builds.

```
2024/08/16 11:58:23 ERRO unable to decode configuration file "xmlto.yaml": yaml: unmarshal errors:
  line 44: field git not found in type config.Update
yamlfile is xmlto.yaml
Building package xmlto with version  from file xmlto.yaml
```

Is there a melange PR to add the right schema support? Or do you want me to do it? where is your code for that git update, and which settings does it support? Or is that not yet finalised?

**Comment by rawlingsj** [comment]
It is in melange and we are already using it in Wolfi.

File: xmlto.yaml [hunk 1]
```diff
@@ -47,6 +48,5 @@ subpackages:
     description: xmlto manpages
 
 update:
-  enabled: true
-  release-monitor:
-    identifier: 13307
+  enabled: false
+  exclude-reason: needs melange schema to support `git` update
```
Comment (rawlingsj) [inline xmlto.yaml:52]:
```suggestion
  enabled: true
  git: {}
```

File: xmlto.yaml [hunk 2]
```diff
@@ -47,6 +48,6 @@ subpackages:
     description: xmlto manpages
 
 update:
+  enabled: false
```
Comment (rawlingsj) [inline xmlto.yaml:51]:
```suggestion
```

**Review by rawlingsj — APPROVED** [review]

**Comment by rawlingsj** [comment]
whoop https://github.com/wolfi-dev/os/pull/26448/checks?check_run_id=28856017943

**Comment by xnox** [comment]
> It is in melange and we are already using it in Wolfi.

........ yes, i should get more coffee...... now actually upgraded local melange and it is working correctly.

**Pull request merged**